src/commands/starchart.ts

```diff
@@ -0,0 +1,19 @@
+import type { Command } from '../index';
+
+export const cmd: Command = {
+  name: 'starchart',
+  desc: 'Shows a plot of the PolyMC repository stars over time',
+  aliases: ['chart'],
+  exec: async (e) => {
+    const sc_img = await fetch('https://starchart.cc/PolyMC/PolyMC.svg')
+      .then((r) => r as Promise<discord.File(r)>)
+      .then((j) => j);
+    embed = discord.Embed()
```

Comment:
A better way would be to put it all in the message, I think

Reply:
Damn. Mo' problems.